### PR TITLE
Adding Missing and Updating: Functions

### PR DESCRIPTION
**Pull request by Neodevils**
It will be ready sooner.

### CODE REVIEW

**Review by Neodevils — COMMENTED** [review]
It looks great to me, I feel tired cause of lacking of sleep. Let me know, if something needs to be fixed.

File: src/functions/usduserrolecount.md [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
 ```javascript
 bot.command({
-name: "roleCount",
-code: `
-$username has $userRoleCount roles!
-`
-})
+  name: "role-count",
+  code: `
+  $username has $userRoleCount roles!
```
Comment (USERSATOSHI) [inline src/functions/usduserrolecount.md]: it's $userRolesCount

File: src/functions/usduserroles.md [hunk 2]
```diff
@@ -1,22 +1,42 @@
...
+## Example
+
 ```javascript
 bot.command({
-name: "userRoles",
-code: `Your Roles: $userRoles[$authorID;names]`
-//Will return, 'role1,role2'
+  name: "user-roles",
+  code: `
+  Your Roles: $userRoles[$authorID;$guildID;name;,]
```
Comment (USERSATOSHI) [inline src/functions/usduserroles.md:36]: change name to id

cause $userRoles returns `@everyone` for name

File: src/functions/usdusersbanned.md [hunk 3]
```diff
@@ -1,31 +1,44 @@
...
-name: "ban-list",
-code: `
-List of banned users:
-$usersBanned[username;, ]
-`
-})
+  name: "users-banened",
+  code: `
+  List of banned users:
+  $usersBanned[$guildID;yes;name;, ]
```
Comment (USERSATOSHI) [inline src/functions/usdusersbanned.md]: change name to username

File: src/functions/usduserswithrole.md [hunk 4]
```diff
@@ -1,36 +1,42 @@
...
 bot.command({
-name: "usersWithRole",
-code: `
-$usersWithRole[$roleID[Developer];|] 
-`
-})
-// Will return: Leref|Ruben
+  name: "users-with-role",
+  code: `
+  $usersWithRole[$roleID[Profossional];697039582922801182;nickname;, ]
```
Comment (USERSATOSHI) [inline src/functions/usduserswithrole.md]: change nickname to displayName

**Review by USERSATOSHI — CHANGES_REQUESTED** [review]

File: src/functions/usduserrolecount.md (hunk 1 above)
Comment (Neodevils) [reply src/functions/usduserrolecount.md]: Fixed.

**Review by USERSATOSHI — APPROVED** [review]
LGTM

**Pull request merged**